harness/attributes/docker.yml

```diff
@@ -13,7 +13,7 @@ attributes:
 
         KEYCLOAK_USER: = @('keycloak.master.admin.username')
         KEYCLOAK_PASSWORD: = @('keycloak.master.admin.password')
-        KEYCLOAK_INTERNAL_URL: http://keycloak:8080/auth
+        KEYCLOAK_INTERNAL_URL: http://keycloak/auth
```

Comment:
this is incorrect at least for dev

Reply:
Modified although in line with slack discussion, I don't know whether this is the "best" way to achieve it.